### PR TITLE
fixed footer

### PR DESCRIPTION
**Pull request by yohanym95**
I added correct links into footer.

### CODE REVIEW

File: assets/content/footer.html [hunk 1]
```diff
@@ -108,26 +108,26 @@ <h5 class="heading">About</h5>
...
                         <li class="nav-item">
-                            <a href="https://sefglobal.org/about.html" class="nav-link">About</a>
+                            <a href="../about/about.html" class="nav-link">About</a>
                         </li>
                         <li class="nav-item">
                             <a href="https://www.youtube.com/channel/UClw7QbeW_FOJz_fDMvjXsJw/" class="nav-link"
                                target="_blank">Portfolio</a>
                         </li>
                         <li class="nav-item">
-                            <a href="https://medium.com/@SustainableEduF" class="nav-link" target="_blank">Blog</a>
```
Comment (gamage1995) [inline assets/content/footer.html]: @yohanym95 this blog link you've updated is yet to have any content right ? i suggest leaving it to redirect to medium until the content is updated.

@jayasanka-sack

**Review by gamage1995 — COMMENTED** [review]
all clear :) thanks for the PR @yohanym95

**Comment by jayasanka-sack** [comment]
@yohanym95 we need to keep absolute links because of the footer is dynamically used in pages.

**Comment by yohanym95** [comment]
ok i will correct it.

**Pull request closed without merging**